ansible/roles/test/tasks/check_testbed_interfaces.yml

```diff
@@ -0,0 +1,57 @@
+- block:
+  - name: Gathering lab graph facts about the device
+    conn_graph_facts: host={{ inventory_hostname }}
+    connection: local
+
+  - name: Fanout hostname
+    set_fact: fanout_switch={{ device_conn['Ethernet0']['peerdevice'] }}
+
+  - name: Check Fanout interfaces
+    local_action: shell ansible-playbook -i lab fanout.yml -l {{ fanout_switch }} --tags check_interfaces_status
+    ignore_errors: yes
+
+  when: 
+    - check_fanout is defined
+
+- block:
+  - name: Get Portchannel status 
+    shell: show interfaces portchannel
+    register: portchannel_status
+    ignore_errors: yes
+
+  - name: Get teamd dump
+    shell: teamdctl '{{ item }}' state dump
```

Comment:
where the output does go?

Reply:
All gathered info goes to the execution console, where it will be extracted if needed.

Reply:
Can we save the output into the variable and output it as
```
debug: var: output.stdout_lines
```

Reply:
Yes, its simple imrovement, but will be done as new PR (this source brunch doesn't exist anymore).

Reply:
To handle this case, I opened another PR, that includes this changes as well
https://github.com/Azure/sonic-mgmt/pull/815